### DISCUSSION TITLE
Missing stdbool.h

### DISCUSSION THREAD

**Discussion by cole-miller** [bpftrace/bpftrace#2378] [Development]
Hi! My bpftrace script `#include`s a header file for the project I'm instrumenting, which in turn has `#include <stdbool.h>`. This runs into the problem that bpftrace can't find `stdbool.h`, which if I understand correctly is a known issue (https://github.com/iovisor/bpftrace/issues/2094#issuecomment-1004751472). Would the fix for that just be to copy `stdbool.h` into this project's `resources` directory from a suitable version of clang?

**Comment by viktormalik** [comment]
Copying the header wouldn't be sufficient, you also need to create the header in `ClangParser::getDefaultHeaders`. You can get inspired by the way the other standard headers are handled now.

If you give it a try, feel free to open a PR :-)
